### DISCUSSION TITLE
disable in-cell edit [Mar 09]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5314] [Discord Threads]
> 🙋‍♂️ **Alsu**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### both are not possible as of now. May I know your use case? By extended view, you mean a popup view?  
#  
> 🙋‍♂️ **Alsu**  
#### For example, if I click on the cell and start editing, it writes changes into the database every time I pause typing, so I get several entries,  I just need one. Is it possible to disable that future or at least make it save on mouse event or keyboard event 'enter'?  
![image0](https://cdn.discordapp.com/attachments/1070415934986133605/1071192179348865174/image.png)  
#  
> 🙋‍♂️ **wkw**  
#### when u pause typing, u mean there will be a new row adding to your db table?  
#  
> 🙋‍♂️ **Alsu**  
#### yes  
#  
> 🙋‍♂️ **wkw**  
#### can u show your project info? if possible, a video would be nice for demonstrating the issue  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1070415934986133605)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
